app/src/main/java/com/github/sdpteam15/polyevents/user/User.kt

```diff
@@ -91,7 +96,8 @@ class User private constructor(override val databaseUser: DatabaseUserInterface)
     }
 
     override fun newProfile(name: String) {
-        val profile = Profile(name, this)
+        val pid = mutableProfileList!!.size
```

Comment:
This is just while we wait the connexion to firebase right? because getting the id on the size of the list if we can remove a profile from the list could be a problem